packages/ui-toolkit/src/components/atoms/Button/button.css

```diff
@@ -61,7 +61,7 @@
 
 .btn51Primary {
   background: var(--primaryClr);
-  color: var(--primaryBg);
+  color: var(--constantWhite);
```

Comment:
what about other buttons like secondary,tertiary?? Do we need to change that also?

Reply:
Those are fine. No need to make changes to that. Just changed the primary and disabled primary CTA